glTF-Toolkit/src/GLTFLODUtils.cpp

```diff
@@ -33,6 +33,16 @@ namespace
         id = (id.empty()) ? "" : std::to_string(std::stoi(id) + offset);
     }
 
+    inline void AddIndexOffset(MeshPrimitive& primitive, const char* attributeName, size_t offset)
+    {
+        // an empty id string indicates that the id is not inuse and therefore should not be updated
+        if (primitive.HasAttribute(attributeName))

```

Comment:
I'm not sure if it's available in the SDK version you have but MeshPrimitive::TryGetAttributeAccessorId may be better than HasAttribute followed by a find (if available)